pandas/tests/reshape/merge/test_merge.py

```diff
@@ -2542,3 +2542,32 @@ def test_mergeerror_on_left_index_mismatched_dtypes():
     df_2 = DataFrame(data=["X"], columns=["C"], index=[999])
     with pytest.raises(MergeError, match="Can only pass argument"):
         merge(df_1, df_2, on=["C"], left_index=True)
+
+
+def test_multiindex_merge_with_unordered_categoricalindex():
```

Comment:
can you parametrized over ordered=True/False (test name can stay the same)

Reply:
can you locate this test near other catorical merge tests

test_merge_categorical
tests_merge_categorical_unordered_equal # maybe a dupe of this one

if not a dupe, then ok to add this test

Reply:
Sure thing. It doesn't look like the test is a duplicate